charts/synthetics-minion/Chart.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: synthetics-minion
+description: New Relic Synthetics Containerized Private Minion (CPM)
+type: application
+
+version: 1.0.16
+appVersion: 3.0.25
```

Comment:
Could you please add other attributes found in the other charts? home, icon, maintainers.  
We publish the charts in the public Helm repo so everyone can use them and that's useful information to have